webapp/src/monacoSnippets.ts

```diff
@@ -253,14 +253,146 @@ function cachedBuiltinCategories(): pxt.Map<BuiltinCategoryDefinition> {
                     },
                     retType: "number"
                 },
+                {
+                    name: "Math.sqrt",
+                    snippetName: "sqrt",
+                    snippet: `Math.sqrt(4)`,
+                    pySnippet: `Math.sqrt(4)`,
```

Comment:
Does capital M `Math` exist in python? Or is this something special that we do?

Reply:
it's a good point, in python proper it is lowercase math https://docs.python.org/3/library/math.html; ours is off the js namespace / capitalized (e.g. `math.cos(x)` is not valid in a makecode python program, where `Math.cos(x)` is)